docs/api/breadcrumbs.md

```diff
@@ -100,9 +100,20 @@ import CSSProps from '@site/static/usage/breadcrumbs/styling/css-props/index.md'
 <CSSProps />
 
 
+## Properties
```

Comment:
`ion-breadcrumbs` and `ion-picker` are still using the inline TOC (I'm assuming you're only migrating ones that already have playgrounds).